### ISSUE TITLE
`XmlRunListener.getFileName(Instrumentation)` is private and not override-able

### ISSUE THREAD

**Issue by okamayana** [schroepf/TestLab#6]
First of all I'd like to thank you massively for your efforts. This is exactly what I need!

However, I noticed that the `XmlRunListener.getFileName(Instrumentation)` method is private and thus not override-able. This is despite your JavaDoc comment on the method, which suggests you intended this method to be override-able by users:

```
/**
* Get a file for the test report. Override this to create different file patterns.
*
* @param instrumentation The current instrumentation with context
* @return A file name to write the report to
*/
```

I can make a small PR to address this if you'd like. Thanks a bunch, again!

**Comment by schroepf** [comment]
Yes, you're right... seems like at some point i changed the method from `protected` to `private`(don't remember why though).

I'll change it back to `protected`.

**Comment by schroepf** [comment]
Will be fixed with PR: #7

**Comment by schroepf** [comment]
version 0.2.1 fixes this issue
